### PR TITLE
Majors

### PR DESCRIPTION
**Pull request by aryamanshodhan**
Completed the documentation for the majors.

### CODE REVIEW

**Review by mattliu-mygit — APPROVED** [review]
Amazing work!

File: docs/uCredit/frontend/components/majors.md [hunk 1]
```diff
@@ -0,0 +1,1494 @@
...
+| `required_credits`     | Number representing the total number of credits required to satisfy the fine requirement.                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `description`          | String representing a brief description of the fine requirement.                                                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (q-andy) [inline docs/uCredit/frontend/components/majors.md]: Is it possible to mention how the descriptions are formatted for fine reqs? For example,

```
''<b>General Physics I</b> <br /> AS.171.101 General Physics: Physical Science Majors I <br /> <i>OR</i> <br /> AS.171.107 General Physics for Physical Sciences Majors I (AL).''
```

Is there a structure to the ``<b>``, ``<br>``, and ``<i>`` tags?

**Review by q-andy — CHANGES_REQUESTED** [review]
Is it possible to include tag structure on the finereq description field?

File: docs/uCredit/frontend/components/majors.md (hunk 1 above)
Comment (aryamanshodhan) [reply docs/uCredit/frontend/components/majors.md]: Just added a line to specify that HTML tags are supported as long as they are all closed properly.

Comment (mattliu-mygit) [reply docs/uCredit/frontend/components/majors.md]: Assuming this convo is resolved.

**Comment by mattliu-mygit** [comment]
Maybe clean up lock files. We only really need package-lock or yarn-lock

**Comment by aryamanshodhan** [comment]
Deleted package-lock.json

**Review by kyang117 — APPROVED** [review]
Great job!

**Pull request merged**